pkgs/development/tools/omnisharp-roslyn/default.nix

```diff
@@ -62,6 +62,21 @@ let
     </configuration>
   '';
 
+  # temporary version patch - this needs to be fixed upstream
+  sdkVersionPatch = writeText "global.json.patch" ''
+diff --git a/global.json b/global.json
+index fe0b480a..a2ce2eeb 100644
+--- a/global.json
++++ b/global.json
+@@ -1,5 +1,5 @@
+ {
+     "sdk": {
+-        "version": "6.0.100"
++        "version": "6.0"
+     }
+ }
+'';
```

Comment:
Why is this not a patch file or a substituteInPlace?

Reply:
I didn’t know of substituteInPlace. 
I’m going to submit a PR upgrading this package soon, I’ll make use of substituteInPlace while I’m at it, thanks.

Reply:
#164761